### ISSUE TITLE
twitter hashtag

### ISSUE THREAD

**Issue by lekoala** [christianv/jquery-lifestream#191]
Is there a way to have a stream based on a hashtag instead of a user?

**Comment by christianv** [comment]
@lekoala that should be possible but you would have to write you're own service/feed. e.g. `twitterhash`
Have a look at https://github.com/christianv/jquery-lifestream/blob/ea8b69a947a340ef64657f150b81c48f95ba9cb2/src/services/twitter.js to get started

**Comment by lekoala** [comment]
yes, I thought about that too. But since you already have a service for twitter by username, I thought that maybe you could also handle an extra parameter for hashtags as well ?

**Comment by christianv** [comment]
@lekoala That's definitely a possible way as well - we just need to make sure it works with the `me` and `example` page: http://christianv.github.io/jquery-lifestream/me/
